docker/nginx-default.conf

```diff
@@ -21,16 +35,30 @@ server {
 server {
     listen 3002;
 
+    # External API
     location / {
         include cors.conf;
         proxy_pass http://node2:3013/;
     }
 
+    # Internal API
     location /internal/ {
+        # Backward compatibility
         include cors.conf;
         proxy_pass http://node2:3113/;
     }
 
+    location /v2/debug {
+        include cors.conf;
+        proxy_pass http://node2:3113/v2/debug;
+    }
+
+    location /v2/key-blocks {
```

Comment:
```suggestion
    location = /v2/key-blocks {
```

Reply:
Did you tested if the `proxy_pass` below still works with the exact match ?

Reply:
No.